lib/cib/cib_attrs.c

```diff
@@ -591,7 +591,7 @@ set_standby(cib_t * the_cib, const char *uuid, const char *scope, const char *st
 
     } else {
         scope = XML_CIB_TAG_NODES;
-        attr_id = crm_strdup_printf("standby-%.256s", uuid);
+        attr_id = crm_strdup_printf("nodes-%.256s-standby", uuid);
```

Comment:
Is there a reason to change this? While it looks better, I'd prefer to keep backward compatibility with any applications linked against an older version of the library.